ansibullbot/utils/extractors.py

```diff
@@ -513,10 +513,15 @@ def docs(self):
         
             self._DOCUMENTATION_RAW = documentation
 
+            # some docstrings don't pass yaml validation with PyYAML >= 4.2
             try:
                 self._DOCSTRING = yaml.load(self._DOCUMENTATION_RAW)
-            except:
-                self._DOCSTRING = {}
+            except yaml.parser.ParserError as e:
+                #logging.debug(e)
+                logging.warning('%s has non-yaml formatted docstrings' % self.filepath)
```

Comment:
Since this block is just copy-paste, you can put both cases into one except section